### PR TITLE
Add foreman_virt_who_configure

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/theforeman/foreman_virt_who_configure
+
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.11
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+#Requires: %{?scl_prefix}rubygem(deface)
```
Comment (ehelms) [inline rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec]: Can we drop this if it is not needed?

File: rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.11
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+#Requires: %{?scl_prefix}rubygem(deface) 
+Requires: %{?scl_prefix}rubygem(katello) >= 3.0
+BuildRequires: foreman-assets
+BuildRequires: foreman-plugin >= 1.11
+#BuildRequires: %{?scl_prefix}rubygem(deface)
```
Comment (ehelms) [inline rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec]: Can we drop this if it is not needed?

File: rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
```
Comment (ehelms) [inline rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec]: Does foreman_virt_who_configure have any c extensions? If not you can simplify this line to prep with an example from https://github.com/theforeman/foreman-packaging/pull/1537/files

Comment (ehelms) [reply rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec]: That is to say, shorten it down to a single use of `gem_install`

Comment (ares) [reply rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec]: I hope I did what you meant

**Comment by ares** [comment]
Thanks @ehelms, would you mind to rereview?

**Comment by ehelms** [comment]
Looks great @ares ! I will merge and build into nightly and follow up with tagging into 3.4.

**Pull request merged**